documentation/examples/prometheus-kubernetes.yml

```diff
@@ -83,6 +82,42 @@ scrape_configs:
     target_label: __metrics_path__
     replacement: /api/v1/nodes/${1}/proxy/metrics
 
+# Scrape config for Kubelet cAdvisor.
+#
+# In Kubernetes 1.7.0 and later, cAdvisor metrics (those whose names begin with
+# 'container_') have been removed from the Kubelet metrics endpoint.  This job
+# scrapes the cAdvisor endpoint to retrieve those metrics.  This should be
+# removed in Kubernetes 1.6 and earlier versions since it will cause the metrics
+# to be scraped twice.
+- job_name: 'kubernetes-cadvisor'
+
+  # Default to scraping over https. If required, just disable this or change to
+  # `http`.
+  scheme: https
+
+  # This TLS & bearer token file config is used to connect to the actual scrape
+  # endpoints for cluster components. This is separate to discovery auth
+  # configuration because discovery & scraping are two separate concerns in
+  # Prometheus. The discovery auth config is automatic if Prometheus runs inside
+  # the cluster. Otherwise, more config options have to be provided within the
+  # <kubernetes_sd_config>.
+  tls_config:
+    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+
+  kubernetes_sd_configs:
+  - role: node
+
+  relabel_configs:
+  - action: labelmap
+    regex: __meta_kubernetes_node_label_(.+)
+  - target_label: __address__
+    replacement: kubernetes.default.svc:443
+  - source_labels: [__meta_kubernetes_node_name]
+    regex: (.+)
+    target_label: __metrics_path__
+    replacement: /api/v1/proxy/nodes/${1}:4194/metrics
```

Comment:
@discostur ah, so that is easy to fix, you have not applied change I have suggested originally as my first commentary https://github.com/prometheus/prometheus/pull/2918#discussion_r126296307

Use `/api/v1/nodes/${1}:4194/proxy/metrics` instead of `/api/v1/proxy/nodes/${1}:4194/metrics`, as the rbac which I referenced above only give `proxy` access on node level.